config/default.yml

```diff
@@ -228,6 +228,10 @@ Rails/BelongsTo:
                   `belongs_to` relations.
   Enabled: true
   VersionAdded: '0.62'
+  Reference:
+    - https://guides.rubyonrails.org/5_0_release_notes.html
+    - https://github.com/rails/rails/pull/18937
+
```

Comment:
Can you remove the redundant blank line?

```suggestion
```

Reply:
Ah. Sorry. I removed the blank line.

Reply:
Can you move the `Reference` to under the `Description`? Note that no move is required for existing `Reference`s.

Reply:
Looks good. moved the Reference.